lib/media/catalog/index.ts

```diff
@@ -60,63 +61,68 @@ export function isCatalog(catalog: any): catalog is Catalog {
 }
 
 export interface Track {
-	kind: string
-	container: string
+	name: string
```

Comment:
There's a few other fields that should be defined here, basically anything with Location = T.

Unfortunately some of these fields will be optional and default to the value in the root. ex.

```typescript
let packaging = catalog.tracks[0].packaging ?? catalog.packing;
```

I think we should automatically populate the track using the default values so you don't have to perform the above every time. But that can be a future PR.